tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -298,6 +299,43 @@ export async function resolveOptions(typeSpecDirectory: string): Promise<Exclude
     return options
 }
 
+export function trySpecifiyApiVersionToGenerateSDKByTypeSpec(typeSpecDirectory: string, apiVersion: string) {
+    if (apiVersion === '') {
+        logger.warn(`No api-version is provided, skip updating tspconfig.yaml.`);
+        return;
+    }
+    
+    const tspConfigPath = path.join(typeSpecDirectory, 'tspconfig.yaml');
```

Comment:
if we could not find the tspconfig.yaml file, shall we continue or throw exception directly?